modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configuring the monitoring of SR-IOV interface traffic
+In order to collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device, the `FlowCollector` `spec.agent.ebpf.privileged` specification value must be `true`. Then, the eBPF agent monitors other network namespaces in addition to the host network namespaces, which are monitored by default. When a pod with a virtual functions (VF) interface is created, a new network namespace is created. With `SRIOVNetwork` policy `IPAM` configurations specified, the VF interface is migrated from the host network namespace to the pod network namespace.
```

Comment:
* Should this have "a" in front"?
* I know people call these different things, but I've never seen "specification value" used. Maybe just say "field"?
* I'm not sure if starting the second sentence with "Then" makes sense, because you didn't tell them an action to take, you just made a statement saying that it has to be true. Maybe if you change the first sentence to say "you must set the .... to `true`." it would make better sense? And it would be more active, so that might be the way to go, if it makes sense. If it doesn't make sense (because maybe it's not this person that set it?) then maybe reconsider replacing "Then" with a different phrase.
* Should virtual function be singular? I see it both ways in the repo

```suggestion
In order to collect traffic from a cluster with a Single Root I/O Virtualization (SR-IOV) device, you must set the `FlowCollector` `spec.agent.ebpf.privileged` field value to `true`. Then, the eBPF agent monitors other network namespaces in addition to the host network namespaces, which are monitored by default. When a pod with a virtual function (VF) interface is created, a new network namespace is created. With `SRIOVNetwork` policy `IPAM` configurations specified, the VF interface is migrated from the host network namespace to the pod network namespace.
```

Reply:
- I do think I'm missing an article here, good catch. 
- People really do call these different things. I think field is good. I think someone has recommended that I specifically say `value` in another peer review but I don't know for sure. 
- I think changing to "you must set the ... to `true`" is a good suggestion, and I can definitely see how it makes more sense with the "Then".
- I think in this case it is singular.